whois-api/src/main/java/net/ripe/db/whois/api/httpserver/JettyBootstrap.java

```diff
@@ -94,12 +109,29 @@ private Server createAndStartServer(final int port) {
         }
     }
 
+    private FilterHolder createDosFilter() throws JmxException, JMException {
```

Comment:
Maybe add a comment linking to https://www.eclipse.org/jetty/documentation/current/dos-filter.html that explains the options?

Reply:
Good suggestion, will add.